presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java

```diff
@@ -60,6 +60,8 @@
     private final String updateType;
     private final Optional<StageInfo> outputStage;
     private final FailureInfo failureInfo;
+    private final Optional<String> failedTaskId;
```

Comment:
Name these `failureTask` and `failureHost` to match `QueryFailureInfo`. (update everywhere in this PR)